LBH.AdultSocialCare.Api/V1/Factories/ResidentialCarePackageFactory.cs

```diff
@@ -81,15 +75,11 @@ public static ResidentialCarePackageResponse ToResponse(ResidentialCarePackageDo
                 IsThisUserUnderS117 = residentialCarePackageDomain.IsThisUserUnderS117,
                 NeedToAddress = residentialCarePackageDomain.NeedToAddress,
                 TypeOfCareHome = residentialCarePackageDomain.TypeOfCareHome,
-                Weekly = residentialCarePackageDomain.Weekly,
-                OneOff = residentialCarePackageDomain.OneOff,
-                AdditionalNeedToAddress = residentialCarePackageDomain.AdditionalNeedToAddress,
                 CreatorId = residentialCarePackageDomain.CreatorId,
-                DateCreated = residentialCarePackageDomain.DateCreated,
                 UpdatorId = residentialCarePackageDomain.UpdatorId,
-                DateUpdated = residentialCarePackageDomain.DateUpdated,
                 StatusId = residentialCarePackageDomain.StatusId,
-                Status = residentialCarePackageDomain.Status
+                Status = residentialCarePackageDomain.Status,
+                ResidentialCareAdditionalNeeds = residentialCarePackageDomain.ResidentialCareAdditionalNeeds
```

Comment:
Consider automapper. It can reduce your workload when such changes happen

Reply:
Will consider!